### PR TITLE
Add a workflow to auto-add @dev-prod issue/PR to projects

### PR DESCRIPTION
**Pull request by blindpirate**

### CODE REVIEW

**Comment by blindpirate** [comment]
@bot-gradle test and merge

Merge for test

File: .github/workflows/add-to-dev-prod-project.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    steps:
+      - name: Get project data
+        run: |
+          gh api graphql -f query='
+            query($org: String!, $number: Int!) {
+              organization(login: $org){
+                projectNext(number: $number) {
+                  id
+                  fields(first:20) {
+                    nodes {
```
Comment (wolfs) [inline .github/workflows/add-to-dev-prod-project.yml:22]: Why include the nodes here at all? You only need the id, right?

File: .github/workflows/add-to-dev-prod-project.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                      id
+                      name
+                      settings
+                    }
+                  }
+                }
+              }
+            }' -f org=gradle -F number=17 > project_data.json
```
Comment (wolfs) [inline .github/workflows/add-to-dev-prod-project.yml:30]:
```suggestion
            }' -F org=gradle -F number=17 > project_data.json
```

Shouldn't this be `F` in both cases?

File: .github/workflows/add-to-dev-prod-project.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      - name: Add issue/PR to project
+        run: |
+          gh api graphql -f query='
+            mutation($project:ID!, $pr:ID!) {
+              addProjectNextItem(input: {projectId: $project, contentId: $pr}) {
+                projectNextItem {
+                  id
+                }
+              }
+            }' -f project=$PROJECT_ID -f pr=${{ github.event.pull_request.node_id }}
```
Comment (wolfs) [inline .github/workflows/add-to-dev-prod-project.yml:42]:
```suggestion
            }' -F project=$PROJECT_ID -F pr=${{ github.event.pull_request.node_id }}
```

Same here, I think this should be `F`.

File: .github/workflows/add-to-dev-prod-project.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+                      id
+                      name
+                      settings
+                    }
+                  }
+                }
+              }
+            }' -f org=gradle -F number=17 > project_data.json
+
+          echo 'PROJECT_ID='$(jq '.data.organization.projectNext.id' project_data.json) >> $GITHUB_ENV
```
Comment (wolfs) [inline .github/workflows/add-to-dev-prod-project.yml:32]: You can use `-q` for `gh api` to pipe the result directly through a `jq` query. I am not sure if it makes the code here easier to read.

**Pull request merged**